### PR TITLE
React - Week#3 BookStore App - Components

### PR DESCRIPTION
**Pull request by mrigorir**
# BookStore

> The Bookstore is an MVP website from [ Zeplin project](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) that offers the following features:

- Display a list of books.

- Add a book.

- Remove a selected book.

## This project is built with: 

- React. 

- React Router.

- Redux. 

- Components.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @mrigorir 👋

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/books/index.jsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const Books = () => <h1>BookStore view</h1>;
```
Comment (mikenath223) [inline src/components/books/index.jsx]: - [ ] The books component is expected to render a list of books using the Book component. For now, you can simply iterate a list of mock books.

File: src/components/books/index.jsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```
Comment (mikenath223) [inline src/components/books/index.jsx]: - [ ] Kindly add a book component that would display data on a single book. It should take the book data as a prop and renders the title and the author.

Comment (mikenath223) [reply src/components/books/index.jsx]: - [ ] In addition, the single book component should also contain a button for removing the book.

File: src/App.js [hunk 3]
```diff
@@ -1,24 +1,22 @@
-import logo from './logo.svg';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import Books from './components/books';
+import Categories from './components/categories';
+import Nav from './components/nav';
```
Comment (mikenath223) [inline src/App.js:3]: - [ ] Kindly add a component that creates books. It should contain a form with inputs for the title and the author of the book. It should also contain a button to submit which would just be a placeholder for now.

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @mrigorir  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**